Example/PageBuilderExtensionSlider/view/adminhtml/web/js/content-type/slider/preview.js

```diff
@@ -0,0 +1,24 @@
+define([
+    'Magento_PageBuilder/js/content-type/slider/preview',
+], function (PreviewBase) {
+    'use strict';
+    var $super;
+
+    function Preview(parent, config, stageId) {
+        PreviewBase.call(this, parent, config, stageId);
+    }
+
+    Preview.prototype = Object.create(PreviewBase.prototype);
+    $super = PreviewBase.prototype;
+
+    Preview.prototype.buildSlickConfig = function buildSlickConfig() {
+        var data = this.contentType.dataStore.getState();
+        var settings = $super.buildSlickConfig.call(this, arguments);
```

Comment:
$super.buildSlickConfig.apply(this, arguments);

Reply:
Also looking at buildSlickConfig function (Magento/PageBuilder/view/adminhtml/web/js/content-type/slider/preview.js) it doesn't accept arguments.  In which file is $super being referenced?  Maybe I need to pull down latest.

Reply:
If it works with `call` then I think we can eliminate arguments reference altogether:
`$super.buildSlickConfig.call(this)`

Reply:
ah, ok, good. wasn't sure about that.